### PR TITLE
Extract the CDN host configuration from the vizjson

### PR DESCRIPTION
**Pull request by javierarce**
Right now we are using the default one, which works in production, but we need to use the one specified in the the vizjson so it works locally and in staging.

### CODE REVIEW

**Comment by javierarce** [comment]
@CartoDB/frontend / @javisantana: would you mind having a look, please?

**Comment by javierarce** [comment]
@CartoDB/frontend: would you mind having a look, please?

File: test/spec/core/image.spec.js [hunk 1]
```diff
@@ -70,6 +70,20 @@ describe("Image", function() {
...
   });
 
+  it("should extract the cdn_url from the vizjson", function(done) {
+
+    var vizjson = "http://documentation.cartodb.com/api/v2/viz/e7b04b62-b901-11e4-b0d7-0e018d66dc29/viz.json";
+
+    var image = cartodb.Image(vizjson);
+
+    image.getUrl(function(err, url) {
+      expect(image.options.cdn_url.http).toEqual("api.cartocdn.com");
```
Comment (xavijam) [inline test/spec/core/image.spec.js]: If the cdn url is different? is it customizable?

Comment (javierarce) [inline test/spec/core/image.spec.js]: That's not the objective of this ticket, but I'll create a new one to allow using a custom CDN :)

**Comment by javisantana** [comment]
we use the cdb_url is set in the layergroup call, we never use viz.json one. Why we decided to do this?

**Comment by javierarce** [comment]
Because it wasn't working and I thought I had to explicitly set it using the one coming from the viz.json.

**Comment by javierarce** [comment]
I'm creating a ticket to fix this.

**Pull request merged**